File: Practices/financial_calculator.py
#AL, Financial Calculator py

monthly_income = float(input("how much money do you earn monthly? "))
rent = float(input("what is your rent? "))
utilities = float(input("what is your monthly utilities? "))
groceries = float(input("what is your monthly groceries? "))
transportation = float(input("what is your montly transportation? "))

total = monthly_income - (rent + utilities + groceries + transportation + (monthly_income * 0.1))

print(f" Your rent is ${rent} and that is {(rent // monthly_income) * 100}%")
print(f" Your utilities are $ {utilities} and that is {(utilities // monthly_income) * 100}% of your income. ")
print(f" Your groceries are $ {groceries} and that is {(groceries // monthly_income) * 100}% of your income. ")
print(f" Your transportation are $ {transportation} and that is {(transportation / monthly_income) * 100}% of your income. ")
print(f" You should save ${monthly_income * 0.1} and that is 10% of your income.")
print(f" You have ${total} of spending money each month! \n")

File: Practices/financial_calculator.c
// AL Financial Calculator 3
#include <stdio.h>

int main() {
    float monthly_income, rent, utilities, groceries, transportation, savings, remaining;
    float rentPercent, utilitiesPercent, groceriesPercent, transportationPercent, savingsPercent;
    
    printf("What's your monthly income:? $");
    scanf("%f", &monthly_income);
    printf("What's your monthly rent?: $");
    scanf("%f", &rent);
    printf("What's your monthly utilities?: $");
    scanf("%f", &utilities);
    printf("What's your monthly groceries?: $");
    scanf("%f", &groceries);
    printf("What's your monthly transportation?: $");
    scanf("%f", &transportation);
    
    savings = monthly_income * 0.10;
    remaining = monthly_income - (rent + utilities + groceries + transportation + savings);
    
    rentPercent = (rent / monthly_income) * 100;
    utilitiesPercent = (utilities / monthly_income) * 100;
    groceriesPercent = (groceries / monthly_income) * 100;
    transportationPercent = (transportation / monthly_income) * 100;
    savingsPercent = (savings / monthly_income) * 100;
    
    printf("\nMonthly Summary\n");
    printf("Savings (10%%): $%.2f\n", savings);
    printf("Remaining to spend: $%.2f\n", remaining);

    printf("\nPercent of Income\n");
    printf("Rent: %.2f%%\n", rentPercent);
    printf("Utilities: %.2f%%\n", utilitiesPercent);
    printf("Groceries: %.2f%%\n", groceriesPercent);
    printf("Transportation: %.2f%%\n", transportationPercent);
    printf("Savings: %.2f%%\n", savingsPercent);

    return 0;
}
